bin/load-latest-fonts.php

```diff
@@ -6,22 +6,11 @@
  * Download latest font packs from their sources.
  */
 
-$packs = array(
-	array(
-		'name' => 'fa',
-		'github-repo' => 'FortAwesome/Font-Awesome',
-		'css-file' => 'css/font-awesome.css',
-
-		'fonts' => array(
-			'fonts/FontAwesome.otf',
-			'fonts/fontawesome-webfont.eot',
-			'fonts/fontawesome-webfont.svg',
-			'fonts/fontawesome-webfont.ttf',
-			'fonts/fontawesome-webfont.woff',
-			'fonts/fontawesome-webfont.woff2'
-		)
-	),
+function get_libs_dir() {
+	return __DIR__ . '/../framework/static/libs/';
```

Comment:
This script should be run by ourselves from time to time, in order to keep packs versions up to date. It is not included in the runtime.